string1.py

```diff
@@ -45,24 +46,32 @@ def donuts(count):
 
 
 def both_ends(s):
-    # your code here
-    return
+    first_two = ''.join(list(s[0:2]))
+    last_two = ''.join(list(s[-2:]))
+    new_word = first_two + last_two
 
+    if len(s) <= 2:
+        return ''
```

Comment:
The if-statement should come first, before you start manipulating the data.  Because what if len(s) < 2?  You will get indexing errors, or you will waste time on the computer doing calculations that are irrelevant